docs/reference/bases.rst

```diff
@@ -0,0 +1,114 @@
+Bases
+=====
+
+.. include:: /reuse/bases-intro.rst
+
+.. _base-snap-reference:
+
+Base snaps
+----------
+
+There are six supported base snaps:
+
+.. list-table::
+   :header-rows: 1
+
+   * - Name
+     - Description
+     - Latest snapcraft track with support
+   * - `core24`_
+     - built from `Ubuntu 24.04 LTS`_
+     - 8.x
+   * - `core22`_
+     - built from `Ubuntu 22.04 LTS`_
+     - 8.x
+   * - `core20`_
+     - built from `Ubuntu 20.04 LTS`_
+     - 8.x
+   * - `core18`_
+     - built from `Ubuntu 18.04 ESM`_
+     - 7.x
+   * - `core`_
+     - built from `Ubuntu 16.04 ESM`_, not to be confused with core16 (see
+       below)
+     - 4.x
+   * - `bare`_
+     - an empty base that's useful with fully statically linked snaps and when
+       testing
+     - 8.x
+
+Older releases of core were occasionally referred to as *core 16*, but ``core``
+and ``core16`` are two distinct packages.
+
+.. warning::
+
+   `core16`_ is not a supported base. With no stable release, its beta and
+   candidate releases are classed as experimental.
+
+See `Snapcraft and ESM`_ for details bases built from ESM releases.
+
+``base``
+--------
+
+The ``base`` keyword in a ``snapcraft.yaml`` file:
+
+* defines the feature set used by Snapcraft
+* the ``snapcraft.yaml`` schema
+* the environment where the snap is built if ``build-base`` is not defined
+* and which base snap is used at runtime
+
+``base`` must be defined except for base and snapd snaps.
```

Comment:
This is a behavioral change from ST119.  Previously, `kernel` snaps also did not have to define `base`.  

We have to define a `base` in order to determine the feature set and snapcraft.yaml schema to use.

Reply:
Updated the reference to include baseless kernels.  I decided not to add a how-to for baseless kernels.

Reply:
we could warn in this scenario and discourage its use in the documentation